core/arch/arm/include/sm/optee_smc.h

```diff
@@ -422,6 +422,44 @@
 #define OPTEE_SMC_VM_DESTROYED \
 	OPTEE_SMC_FAST_CALL_VAL(OPTEE_SMC_FUNCID_VM_DESTROYED)
 
+/*
+ * Query OP-TEE about its build configuration
+ *
+ * Normal World OS or Hypervisor issues this call to find out certain OP-TEE
+ * build options, like number of supported threads or if virtualization support
+ * is enabled.
+ *
+ * Call requests usage:
+ * a0	SMC Function ID, OPTEE_SMC_GET_CONFIG
+ * a1	Identifier of configuration option
+ * a2-6 Not used
+ * a7	Hypervisor Client ID register
+ *
+ * Normal return register usage:
+ * a0	OPTEE_SMC_RETURN_OK
+ * a1-6	Requested information
+ * a7	Preserved
+ *
+ * Error return:
+ * a0	OPTEE_SMC_RETURN_ENOTAVAIL	Unknown configuration option was
+ *					supplied in a1 register
+ * a1-7	Preserved
+ */
+#define OPTEE_SMC_FUNCID_GET_CONFIG	15
+#define OPTEE_SMC_GET_CONFIG \
+	OPTEE_SMC_FAST_CALL_VAL(OPTEE_SMC_FUNCID_GET_CONFIG)
+
+/* Possible configuration options for OPTEE_SMC_FUNCID_GET_CONFIG call: */
+
+/* a1 will contain 0. Used to check OPTEE_SMC_FUNCID_GET_CONFIG availability */
+#define OPTEE_SMC_CONFIG_NONE		0
```

Comment:
Not needed, if it's not available you'll get `OPTEE_SMC_RETURN_UNKNOWN_FUNCTION` which I guess is good enough.